conda_smithy/templates/appveyor.yml.tmpl

```diff
@@ -44,23 +38,17 @@ install:
         Where-Object pullRequestId -eq $env:APPVEYOR_PULL_REQUEST_NUMBER)[0].buildNumber) { `
           throw "There are newer queued builds for this pull request, failing early." }
 
-    # Cywing's git breaks conda-build. (See https://github.com/conda-forge/conda-smithy-feedstock/pull/2.)
-    - cmd: rmdir C:\cygwin /s /q
```

Comment:
I think this comment is actually out of date - I believe AppVeyor changed the default PATH so that this git wasn't picked up by default, so we should be safe to remove it.

You raise a really interesting question about `conda-forge-build-setup` - it already knows some things about TravisCI, which lends support for it knowing about AppVeyor, but I do wonder if we can avoid the CI knowledge without loosing the fantastic flexibility that `conda-forge-build-setup` can give us.